examples/multiple_clusters.py

```diff
@@ -12,39 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Allows you to pick a context and then lists all pods in the chosen context.
+
+Please install the pick library before running this example.
+"""
+
 from kubernetes import client, config
-# install pick using "pip install pick". It is not included
-# as a dependency because it only used in examples
-from pick import pick
+from kubernetes.client import configuration
+from pick import pick  # install pick using `pip install pick`
 
 
 def main():
-
     contexts, active_context = config.list_kube_config_contexts()
     if not contexts:
         print("Cannot find any context in kube-config file.")
         return
     contexts = [context['name'] for context in contexts]
     active_index = contexts.index(active_context['name'])
-    cluster1, first_index = pick(contexts, title="Pick the first context",
-                                 default_index=active_index)
-    cluster2, _ = pick(contexts, title="Pick the second context",
-                       default_index=first_index)
-
-    client1 = client.CoreV1Api(
-        api_client=config.new_client_from_config(context=cluster1))
-    client2 = client.CoreV1Api(
-        api_client=config.new_client_from_config(context=cluster2))
-
-    print("\nList of pods on %s:" % cluster1)
-    for i in client1.list_pod_for_all_namespaces().items:
-        print("%s\t%s\t%s" %
-              (i.status.pod_ip, i.metadata.namespace, i.metadata.name))
-
-    print("\n\nList of pods on %s:" % cluster2)
-    for i in client2.list_pod_for_all_namespaces().items:
-        print("%s\t%s\t%s" %
-              (i.status.pod_ip, i.metadata.namespace, i.metadata.name))
+    option, _ = pick(contexts, title="Pick the context to load",
```

Comment:
This has changed the original purpose of the example, which is for multiple clusters. The modified example looks to be for a single cluster

Reply:
I agree. @scottilee did you accidentally copied the content of `pick_kube_config_context.py` (originally `example4.py`) to `multiple_clusters.py`?

Reply:
Reverted.